shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/OverlaySchedule.hs

```diff
@@ -0,0 +1,184 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE LambdaCase #-}
+
+module Shelley.Spec.Ledger.OverlaySchedule
+  ( -- * Overlay schedule
+    OverlaySchedule,
+    compactOverlaySchedule,
+    decompactOverlaySchedule,
+    emptyOverlaySchedule,
+    isOverlaySlot,
+    lookupInOverlaySchedule,
+    overlaySchedule,
+    overlayScheduleHelper,
+    overlayScheduleIsEmpty,
+    overlayScheduleToMap,
+
+    -- * OBftSlot (for testing)
+    OBftSlot (..),
```

Comment:
Not just for testing, it's mentioned in the public API, i.e., `lookupInOverlaySchedule`.

Reply:
Yep, my search didn't spot it, then I realised I had a broken search...